wn/_add.py

```diff
@@ -138,10 +138,16 @@ def _add_lmf(
         _update_lookup_tables(all_infos, cur)
 
         progress.flash(f'Reading {source!s}')
-        for lexicon, info in zip(lmf.load(source), all_infos):
+
+        total_items = sum(_sum_counts(info) for info in all_infos)
+        progress.set(count=0,
+                     total=total_items,
+                     refresh_interval=50,
+                     status='Reading XML')
```

Comment:
Not exactly what I meant. I meant to use the same `ProgressHandler` class to create the `progress_lmf` instance. But let's just leave this for now as it's not so bad to reuse the same progress bar.

Reply:
addressed. I instantiated using the same class, and created another progress handler for the lmf load.